tasks/setup_plugins.yml

```diff
@@ -21,6 +21,8 @@
     update: True
   tags: [ 'role::checkmk_agent:plugins:get' ]
   delegate_to: '{{ checkmk_agent__git_dest_host }}'
+  become: '{{ False if (checkmk_agent__git_dest_host in ["localhost"]) else True }}'
+  become_user: '{{ lookup("env","USER") if (checkmk_agent__git_dest_host in ["localhost"]) else ansible_become_user|d("root") }}'
```

Comment:
Because `become_user` might be defined differently on the playbook level which would not work when delegated to localhost.